node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -196,7 +193,7 @@ where
         let consensus_heap_size = {
             let any_ref = consensus.as_any();
 
-            if let Some(highway) = any_ref.downcast_ref::<HighwayProtocol<I, ClContext>>() {
+            if let Some(highway) = any_ref.downcast_ref::<HighwayProtocol<ClContext>>() {
```

Comment:
Ideally, `HighwayProtocol` should still abstract away `NodeId`. It's strictly networking layer type.

Reply:
When you say "its strictly networking layer type", do you mean that, "up to now, it has strictly been a networking layer type", or do you mean that "there are structural reasons why it _must_ be a network only type"?

Reply:
I'm saying that it's an abstraction/type/decision of the networking layer and its details are not important for abstraction of consensus protocol.